integration/ramdisk/ramdisk.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+cidir=$(dirname "$0")
+
+source "${cidir}/../../lib/common.bash"
+
+set -e
+
+# Environment variables
+IMAGE="${IMAGE:-busybox}"
+CONTAINER_NAME="${CONTAINER_NAME:-test}"
+PAYLOAD_ARGS="${PAYLOAD_ARGS:-sleep 30}"
+
+# Set the runtime if not set already
+RUNTIME="${RUNTIME:-kata-runtime}"
+
+# File to save ramdisk results
+TMP_FILE=$(mktemp ramdisk.XXXXXXXXXX)
+
+remove_tmp_file() {
+	rm -rf $TMP_FILE
+}
+
+trap remove_tmp_file EXIT
+
+setup() {
+	clean_env
+
+	# Stop docker
+	sudo -E systemctl stop docker
+}
+
+test_ramdisk() {
+	# Grab the time (filter journalctl)
+	grab_time=$(date +"%H:%M:%S")
+
+	# Enable ramdisk with dockerd
+	(sudo -E DOCKER_RAMDISK=true dockerd -D --add-runtime ${RUNTIME}=${RUNTIME_PATH} 2>/dev/null) &
+
+	# Run container
+	docker run --rm -d --name=${CONTAINER_NAME} --runtime=${RUNTIME} ${IMAGE} sh -c ${PAYLOAD_ARGS}
```

Comment:
Saw this at the log:
```
docker: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?.
```

It seems that `dockerd` is not ready when calling `docker run ...`, you'll need to wait until `dockerd` is ready to receive requests.